Library/Formula/git-polish-history.rb

```diff
@@ -0,0 +1,33 @@
+class GitPolishHistory < Formula
+  homepage "https://github.com/schani/git-polish-history"
+  url "https://github.com/schani/git-polish-history.git", :tag => "v0.1"
+
+  depends_on "go" => :build
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "openssl" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    # Install Go dependencies
+    system "go", "get", "-d", "github.com/schani/git2go"
+    cd "#{buildpath}/src/github.com/schani/git2go" do
+      system "git", "submodule", "update", "--init"
+      system "./script/build-libgit2-static.sh"
+      system "go", "run", "script/check-MakeGitError-thread-lock.go"
+      system "./script/with-static.sh", "go", "test", "./..."
+      system "./script/with-static.sh", "go", "install", "./..."
+    end
+
+    system "go", "get", "github.com/codegangsta/cli"
```

Comment:
You should use `go_resource` otherwise you're not using versioned dependencies and this could break at any time when the projects change their `master` branches.